cloudsmith_cli/cli/commands/auth.py

```diff
@@ -16,18 +16,20 @@
 AUTH_SERVER_PORT = 12400
 
 
-def _perform_saml_authentication(opts, owner, enable_token_creation=False):
+def _perform_saml_authentication(opts, owner, enable_token_creation=False, json=False):
     """Perform SAML authentication via web browser and local web server."""
     session = create_configured_session(opts)
     api_host = opts.api_config.host
 
     idp_url = get_idp_url(api_host, owner, session=session)
-    click.echo(
-        f"Opening your organization's SAML IDP URL in your browser: {click.style(idp_url, bold=True)}"
-    )
+    if not json:
+        click.echo(
```

Comment:
we have existing patterns for handling stdout/stderr depending on the output format, are those more appropriate here? e.g. https://github.com/cloudsmith-io/cloudsmith-cli/blob/b145d35f154c233970f199ba8497304920f0282b/cloudsmith_cli/cli/commands/list_.py#L55